authzed/api/v1/acl_service.proto

```diff
@@ -33,9 +45,22 @@ service PermissionsService {
 // requirements should be applied, where applicable.
 message Consistency {
   oneof requirement {
+    // minimize_latency indicates that the latency for the call should be minimized by having
+    // the system select the fastest snapshot available
     bool minimize_latency = 1 [ (validate.rules).bool.const = true ];
+
+    // at_least_as_fresh indicates that all data used in the API call must be *at least as fresh*
+    // as that found in the ZedToken; more recent data might be used if available or faster.
     ZedToken at_least_as_fresh = 2;
+
+    // at_exact_snapshot indicates that all data used in the API call must be *at the given*
+    // snapshot in time; if the snapshot is no longer available, an error will be returned to
+    // the caller.
     ZedToken at_exact_snapshot = 3;
+
+    // fully_consistent indicates that all data used in the API call *must* be at the most recent
+    // snapshot found. NOTE: using this method can be *quite slow*, so unless there is a need to do
+    // so, it is recommended to use `at_least_as_fresh` with a stored ZedToken.
```

Comment:
break the note into its own paragraph so that it stands out

```proto
    // fully_consistent indicates that all data used in the API call *must* be at the most recent
    // snapshot found.
    //
    // NOTE: using this method can be *quite slow*, so unless there is a need to do
    // so, it is recommended to use `at_least_as_fresh` with a stored ZedToken.
```